### ISSUE TITLE
[BUG] configure network failed if use static mode

### ISSUE THREAD

**Issue by futuretea** [harvester/harvester#2796]
**Describe the bug**

![image](https://user-images.githubusercontent.com/15064560/191430124-c798f203-06a1-469b-9152-75f2a4d9b537.png)
![image](https://user-images.githubusercontent.com/15064560/191430144-989603a0-19e0-4fba-9af8-7d4ef4011fff.png)

**To Reproduce**
1. use the latest master-head iso 
2. set ipv4 mode to Static
3. set ip address and gateway

**Expected behavior**

configure network successfully

**Support bundle**

**Environment**
 - Harvester ISO version: 
 - Underlying Infrastructure (e.g. Baremetal with Dell PowerEdge R630):

**Additional context**

**Comment by tjjh89017** [comment]
we will test this issue in 1390 & 1647 at the same time and skip the test in this issue.
https://github.com/harvester/harvester/issues/1390
https://github.com/harvester/harvester/issues/1647

**Comment by futuretea** [comment]
also has this issue when keeping vlan id empty
![image](https://user-images.githubusercontent.com/15064560/191431905-dd47e3ea-3d89-4913-a17f-46b25226e915.png)

![image](https://user-images.githubusercontent.com/15064560/191434328-4f40a1d3-767a-4189-94dc-15d66f5072a9.png)

**Comment by bk201** [comment]
I remove the `not-require/test-plan` label, this deserves an issue in `harvester/tests`.

**Comment by tjjh89017** [comment]
reproduce/test step

## VLAN empty

1. use latest ISO to install
2. skip VLAN field
3. choose static method
4. fill other fields
5. press enter to the next page
6. no error found, and show DNS config page


## VLAN 1

1. use latest ISO to install
2. set VLAN 1
3. choose static method
4. fill other fields
5. press enter to the next page
6. no error found, and show DNS config page

## VLAN other

1. use latest ISO to install
2. set VLAN 1000 other than empty, 0, 1
3. choose static method
4. fill other fields
5. press enter to the next page
6. no error found, and show DNS config page

**Comment by TachunLin** [comment]
Verified **fixed** on `master-250f41e4-head` (0921 22:00 CST). Close this issue.

### Result
During Harvester ISO installer 
We can configure VLAN network on the static mode with the following settings: 
1. No error message blocked
2. Can proceed to dns config page 
 
#### VLAN Empty
https://user-images.githubusercontent.com/29251855/191526917-9336dc72-e52c-43dc-aee2-cdb00910094c.mp4

![image](https://user-images.githubusercontent.com/29251855/191526937-70e24912-d661-4c90-969b-642552293b69.png)


#### VLAN 1
https://user-images.githubusercontent.com/29251855/191527929-3b6cc568-b182-42eb-9ce6-f0684a5a3adc.mp4

![image](https://user-images.githubusercontent.com/29251855/191527878-471fb808-55a1-4f35-84e0-b8cf089b6182.png)


#### VLAN 1000 (Other)
https://user-images.githubusercontent.com/29251855/191529094-b2d01978-9038-472d-83ba-4ac2e8839aae.mp4

### Test Information
* Test Environment: 1 node Harvester on local kvm machine
* Harvester version: `master-250f41e4-head` (0921 22:00 CST)

### Verify Steps

1. Use latest ISO to install
2. Enter VLAN field with 
 - empty
 - 1
 - 1000
2. choose static method
3. fill other fields
4. press enter to the next page
5. no error found, and show DNS config page

### Additional Context
* DHCP lease of `default` VM virtual network (No occupation)
```
davidtclin@localhost:~> virsh net-dhcp-leases default
 Expiry Time           MAC address         Protocol   IP address           Hostname       Client ID or DUID
----------------------------------------------------------------------------------------------------------------
 2022-09-21 22:32:24   0e:a8:e8:5c:bd:bb   ipv4       192.168.122.26/24    -              -
 2022-09-21 22:38:58   52:54:00:2a:90:ae   ipv4       192.168.122.230/24   ubuntu-clean   01:52:54:00:2a:90:ae

```
